### PR TITLE
Remove BrightnESS namespace and rename namespace to Forwarder

### LINKED ISSUE #95: Remove/re-name namespaces
Remove the `BrightnESS` namespace and (possibly) re-name the `ForwardEpicsToKafka` namespace to something shorter (`Forwarder`?).

### PR DESCRIPTION
**Pull request by rerpha**
### Description of work

Removed the `BrightnESS` namespace from everywhere and renamed the `ForwardEpicsToKafka` namespace to `Forwarder`

### Issue

Closes #95 

### Acceptance Criteria

- Changed cli11 version in conan file as it was playing up on bintray

### Unit Tests

Changed the namespace in the unit tests

### Other

---

## Code Review (To be filled in by the reviewer only)

- [x] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?

---

## Nominate for Group Code Review (Anyone can nominate it)
Indicate if you think the code should be reviewed in a Thursday code review session.

- [ ] Recommend for group code review

Also, nominate it on the code_review Slack channel (does someone want to automate this?).

### CODE REVIEW

File: src/FlatbufferMessageSlice.h [hunk 1]
```diff
@@ -3,7 +3,6 @@
 #include <cstddef>
 #include <cstdint>
 
-namespace BrightnESS {
 namespace FlatBufs {
```
Comment (mattclarke) [inline src/FlatbufferMessageSlice.h]: Should this be Forwarder::FlatBufs for consistency?

File: src/SchemaRegistry.cpp [hunk 2]
```diff
@@ -1,6 +1,5 @@
 #include "SchemaRegistry.h"
 
-namespace BrightnESS {
 namespace FlatBufs {
```
Comment (mattclarke) [inline src/SchemaRegistry.cpp]: See previous comment

File: src/Streams.cpp [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #include "Streams.h"
 #include "Stream.h"
 
-using namespace BrightnESS::ForwardEpicsToKafka;
+using namespace Forwarder;
```
Comment (mattclarke) [inline src/Streams.cpp]: This is different to the other files which tend to use `namespace Forwarder {`.
LLVM says using `using namespace Forwarder` is okay for .cpp files, as it reduces indentation.
I think we should change the others to match this.

Comment (matthew-d-jones) [reply src/Streams.cpp]: The LLVM format doesn't use indentation for namespaces, so the reasoning seems strange. They are also not equivalent, there are some uses `extern` for example where we'd end up with both bits of syntax. As it is not as straightforward as just swapping syntax I suggest it goes in a different PR if we make this change. If LLVM doesn't dictate that you must do it that way, I'm not convinced it is worth the time.

Comment (mattclarke) [reply src/Streams.cpp]: Fair enough, let's change this one to match the others and be done with it.

**Pull request merged**